coordinator/src/validators/request-validator.ts

```diff
@@ -16,61 +18,89 @@ export function verifyPrescriptionBundle(bundle: unknown, requireSignature: bool
     return [new errors.NoEntryInBundleError()]
   }
 
+  const bundleValidationErrors = verifyCommonErrors(bundle, requireSignature)
+
+  const messageHeaderError = verifyBundleContainsExactly(bundle, 1, "MessageHeader")
+  if (messageHeaderError) {
+    return [messageHeaderError, ...bundleValidationErrors]
+  }
+
+  // bit annoying that we call identifyMessageType on line 28 (through verifyTypeOfBundle) and on line 32
```

Comment:
Either change it or remove the comment. Don't think it's a big deal to leave it as is